renderer/components/onboard/__tests__/QuizSteps.test.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @jest-environment jsdom
+ */
+import React from 'react'
+import {
+  screen,
+  render,
+  fireEvent,
+  waitForElementToBeRemoved,
+} from '@testing-library/react'
+import { theme } from 'ooni-components'
+import { ThemeProvider } from 'styled-components'
+import { IntlProvider, FormattedMessage } from 'react-intl'
+import English from '../../../../lang/en.json'
+
+import QuizSteps from '../QuizSteps'
+
+const renderComponent = (component, locale = 'en', messages = English) => {
+  return render(
+    <IntlProvider locale={locale} messages={messages}>
+      <ThemeProvider theme={theme}>{component}</ThemeProvider>
+    </IntlProvider>
+  )
+}
+
+describe('Tests for QuizSteps', () => {
+  const toggleQuiz = jest.fn()
+  const onQuizComplete = jest.fn()
+
+  beforeEach(() => {
+    renderComponent(
+      <QuizSteps
+        onClose={toggleQuiz}
+        onDone={onQuizComplete}
+        questionList={[
+          <FormattedMessage
+            key="Onboarding.PopQuiz.1.Question"
+            id="Onboarding.PopQuiz.1.Question"
+          />,
+          <FormattedMessage
+            key="Onboarding.PopQuiz.2.Question"
+            id="Onboarding.PopQuiz.2.Question"
+          />,
+        ]}
+        actuallyList={[
+          <FormattedMessage
+            key="Onboarding.PopQuiz.1.Wrong.Paragraph"
+            id="Onboarding.PopQuiz.1.Wrong.Paragraph"
+          />,
+          <FormattedMessage
+            key="Onboarding.PopQuiz.2.Wrong.Paragraph"
+            id="Onboarding.PopQuiz.2.Wrong.Paragraph"
+          />,
+        ]}
+      />
+    )
+  })
+
+  afterEach(() => {
+    onQuizComplete.mockClear()
+  })
+
+  test('User Story with correct answers', async () => {
+    const yesButton1 = screen.getByText(English['Onboarding.PopQuiz.True'])
+    fireEvent.click(yesButton1)
+    const lottiePlayerFirst = screen.getByTestId('quiz-steps-animation')
```

Comment:
This doesn't differentiate between `tickAnimation` and `crossAnimation`. Maybe the `data-testid` can be assigned dynamically inside the `<Animation>` component and it can be tested here (and the in the next test case)

Reply:
Yes, you are correct. Thank you for your suggestion! The test now covers both tick and cross animations.